ngraph/core/src/op/deformable_convolution.cpp

```diff
@@ -195,6 +196,136 @@ std::shared_ptr<Node>
+                                                          out->get_data_ptr<ET>(),
+                                                          inputs[0]->get_shape(),
+                                                          inputs[1]->get_shape(),
+                                                          inputs[2]->get_shape(),
+                                                          inputs[3]->get_shape(),
+                                                          out->get_shape(),
+                                                          strides,
+                                                          dilations,
+                                                          pads_begin,
+                                                          pads_end,
+                                                          group,
+                                                          deformable_group,
+                                                          use_bilinear_interpolation_padding);
+        }
+
+        return true;
+    }
+
+    bool evaluate_deformable_convolution(const HostTensorVector& inputs,
+                                         const HostTensorPtr& out,
+                                         const Strides& strides,
+                                         const Strides& dilations,
+                                         const CoordinateDiff& pads_begin,
+                                         const CoordinateDiff& pads_end,
+                                         const ngraph::op::PadType& auto_pad,
+                                         const int64_t group,
+                                         const int64_t deformable_group,
+                                         const bool use_bilinear_interpolation_padding)
+    {
+        bool rc = true;
+        switch (inputs[0]->get_element_type())
+        {
+            NGRAPH_TYPE_CASE(evaluate_deformable_convolution,
+                             f32,
+                             inputs,
+                             out,
+                             strides,
+                             pads_begin,
+                             pads_end,
+                             dilations,
+                             auto_pad,
+                             group,
+                             deformable_group,
+                             use_bilinear_interpolation_padding);
+            NGRAPH_TYPE_CASE(evaluate_deformable_convolution,
+                             f16,
+                             inputs,
+                             out,
+                             strides,
+                             pads_begin,
+                             pads_end,
+                             dilations,
+                             auto_pad,
+                             group,
+                             deformable_group,
+                             use_bilinear_interpolation_padding);
+        default: rc = false; break;
```

Comment:
Looks like not covered by tests

Reply:
"evaluate" method will be deleted after merge cpu plugin implementation
this method has been implemented for testing purposes, e.g. launching models from paddlepaddle